website/content/en/preview/faq.md

```diff
@@ -39,63 +37,46 @@ Yes, as long as the controller has network and IAM/RBAC access to the Kubernetes
-You can manage all capacity using provisioners, use a mixed model with dynamic and statically managed capacity, or use a fully static approach.
-We expect most users will use a mixed approach in the near term and provisioner-managed in the long term.
+NodePools are designed to work alongside static capacity management solutions like EKS Managed Node Groups and EC2 Auto Scaling Groups. You can manage all capacity using NodePools, use a mixed model with dynamic and statically managed capacity, or use a fully static approach. We expect most users will use a mixed approach in the near term and NodePool-managed in the long term.
 
 
 ### How does Karpenter interact with Kubernetes features?
-* Kubernetes Cluster Autoscaler: Karpenter can work alongside cluster autoscaler.
-See [Kubernetes cluster autoscaler]({{< ref "./concepts/#kubernetes-cluster-autoscaler" >}}) for details.
-* Kubernetes Scheduler: Karpenter focuses on scheduling pods that the Kubernetes scheduler has marked as unschedulable.
-See [Scheduling]({{< ref "./concepts/scheduling" >}}) for details on how Karpenter interacts with the Kubernetes scheduler.
+* Kubernetes Cluster Autoscaler: Karpenter can work alongside cluster autoscaler. See [Kubernetes cluster autoscaler]({{< ref "./concepts/#kubernetes-cluster-autoscaler" >}}) for details.
+* Kubernetes Scheduler: Karpenter focuses on scheduling pods that the Kubernetes scheduler has marked as unschedulable. See [Scheduling]({{< ref "./concepts/scheduling" >}}) for details on how Karpenter interacts with the Kubernetes scheduler.
 
 ## Provisioning
 
-### What features does the Karpenter provisioner support?
-See [Provisioner API]({{< ref "./concepts/nodepools" >}}) for provisioner examples and descriptions of features.
+### What features does the Karpenter NodePool support?
+See the [NodePool API docs]({{< ref "./concepts/nodepools" >}}) for NodePool examples and descriptions of features.
 
-### Can I create multiple (team-based) provisioners on a cluster?
-Yes, provisioners can identify multiple teams based on labels.
-See [Provisioner API]({{< ref "./concepts/nodepools" >}}) for details.
+### Can I create multiple (team-based) NodePools on a cluster?
+Yes, NodePools can identify multiple teams based on labels. See the [NodePool API docs]({{< ref "./concepts/nodepools" >}}) for details.
 
-### If multiple provisioners are defined, which will my pod use?
+### If multiple NodePools are defined, which will my pod use?
 
-Pending pods will be handled by any Provisioner that matches the requirements of the pod.
-There is no ordering guarantee if multiple provisioners match pod requirements.
-We recommend that Provisioners are setup to be mutually exclusive.
-Read more about this recommendation in the [EKS Best Practices Guide for Karpenter](https://aws.github.io/aws-eks-best-practices/karpenter/#create-provisioners-that-are-mutually-exclusive).
```

Comment:
Is there a replacement blog in the works?

Reply:
I'm not sure that we have this recommendation anymore. I'll follow-up with @billrayburn on getting the EKS Best Practices guide upgraded separately